### ISSUE TITLE
Qualified infix sections get mangled

### ISSUE THREAD

**Issue by ttuegel** [commercialhaskell/hindent#273]
Given the following input,

``` .haskell
import qualified Data.Vector as V

main :: IO ()
main = do
    let _ = foldr1 (V.++) [V.empty, V.empty]
    pure ()
```

hindent produces this output:

``` .haskell
import qualified Data.Vector as V

main :: IO ()
main = do
    let _ = foldr1 V.(++) [V.empty, V.empty]
    pure ()
```

which is **not** valid Haskell. The section of a qualified infix operator `(V.++)` has been mangled into `V.(++)`.

**Comment by tfausak** [comment]
I don't think this is a problem anymore.

```
tmp> cat Example.hs 
import qualified Data.Vector as V

main :: IO ()
main = do
    let _ = foldr1 (V.++) [V.empty, V.empty]
    pure ()
tmp> hindent Example.hs 
tmp> cat Example.hs 
import qualified Data.Vector as V

main :: IO ()
main = do
  let _ = foldr1 (V.++) [V.empty, V.empty]
  pure ()
```

**Comment by sighingnow** [comment]
Close as solved in #295 .
